iphone/Classes/TiUINavBarButton.m

```diff
@@ -119,6 +119,7 @@ - (id)initWithProxy:(TiUIButtonProxy *)proxy_
       self = [super initWithImage:theimage style:[self style:proxy_] target:self action:@selector(clicked:)];
     } else {
       self = [super initWithTitle:[self title:proxy_] style:[self style:proxy_] target:self action:@selector(clicked:)];
+      self.tintColor = [TiUtils colorValue:[proxy_ valueForKey:@"color"]].color;
```

Comment:
What if no `color` value is given?

Reply:
Again, as mentioned in the ticket: Buttons are tinted via the `tintColor` property, not `color`. Not sure about Android (likely a parity issue there).

Reply:
@hansemannn  'color' property is used to set title color in UIButton. If we set 'tintColor', it will set the tintColor of UIButton but effect is   shown on title of button (meaning color of button's title is changed). If we set 'tintColor' and 'color' both, 'color' will override the effect of 'tintColor'. 
I think we should set 'color' if available otherwise 'tintColor'  for self.tintColor here.

Reply:
@sgtcoolguy In case of no color or tintColor, it will set nil which is default.

Reply:
If you are sure that that's the cross platform behavior, I'm fine with that :-) But in our app, we only use tintColor and it works very well - thats why I wondered.